root/static/scripts/release/components/TrackRelationshipEditor.js

```diff
@@ -244,12 +245,41 @@ component _RelatedWorkRelationshipEditor(
   ]);
 
   const removeWork = React.useCallback(() => {
+    if (isNewWork) {
+      let hasActiveRelationships = false;
+      for (
+        const relationship of
+        iterateRelationshipsInTargetTypeGroups(relatedWork.targetTypeGroups)
+      ) {
+        if (relationship._status === REL_STATUS_REMOVE) {
+          continue;
+        }
+        if (
+          relationship.entity0.entityType === 'recording' &&
+        relationship.entity0.id === track.recording.id &&
+        relationship.entity1.entityType === 'work' &&
+        relationship.entity1.id === relatedWork.work.id
```

Comment:
Very minor (and not sure why ESLint didn't complain), but can you indent these with the first line? :)

```suggestion
          relationship.entity0.entityType === 'recording' &&
          relationship.entity0.id === track.recording.id &&
          relationship.entity1.entityType === 'work' &&
          relationship.entity1.id === relatedWork.work.id
```

Reply:
oops. not sure why Eslint didnt complain. shouldve noticed it myself though :D.